crates/noirc_evaluator/src/brillig/brillig_ir.rs

```diff
@@ -853,11 +853,13 @@ impl BrilligContext {
     pub(crate) fn radix_instruction(
         &mut self,
         source: RegisterIndex,
+        target_vector_len: RegisterIndex,
         target_vector: HeapVector,
         radix: RegisterIndex,
         limb_count: RegisterIndex,
         big_endian: bool,
     ) {
+        self.mov_instruction(target_vector_len, limb_count);
```

Comment:
I think this is an ssa implementation detail so probably the handling of the slice length (mov) could be done outside (I think in brillig_gen, since it's the one that is aware of the ssa implementation details)

Reply:
good call I will move it